addons/base_iban/__manifest__.py

```diff
@@ -11,6 +11,7 @@
 with a single statement.
     """,
     'depends': ['account', 'web'],
+    'auto_install': True,
```

Comment:
Don't auto-install another module for everyone for something unrelated.
Maybe you can change the hr_payroll_account auto_install to ['hr_payroll', 'account_accountant'],